core/unit_test/TestViewMemoryAccessViolation.hpp

```diff
@@ -120,7 +120,7 @@ void test_view_memory_access_violations_from_host() {
   test_view_memory_access_violation(make_view<V6>(lbl), host_exec_space, prefix + ".*" + lbl);
   test_view_memory_access_violation(make_view<V7>(lbl), host_exec_space, prefix + ".*" + lbl);
   test_view_memory_access_violation(make_view<V8>(lbl), host_exec_space, prefix + ".*" + lbl);
-  auto* const ptr = reinterpret_cast<int*>(0xABADBABE);
+  int* const ptr = nullptr;
```

Comment:
Per Daniel's suggestion on Slack
```suggestion
  auto* const ptr = reinterpret_cast<int*>(0x0BADBABE);
```